src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static final int ITEMS = 25;
+    static final int LINES = 85;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
         System.out.println("Hello from\n" + logo);
+
+        for (int i = 0; i < LINES; i++) {
+            System.out.print("_");
+        }
+
+        System.out.print("\n");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I  do for you?");
+
+        for (int i = 0; i < LINES; i++) {
+            System.out.print("_");
+        }
+
+        System.out.print("\n");
+        Scanner scan = new Scanner(System.in);
+
+        //new String[numRowsofData]
+        String[] tasks= new String[ITEMS];
+        String[] checks = new String[ITEMS];
+        String[] type=new String[ITEMS];
+        String[] deadline=new String[ITEMS];
+        String[] parts;
+
+        int index=0;
+        int count;
+        int num;
+        int temp;
+
+        while(true) {
+
+            tasks[index] = scan.nextLine();
+            checks[index] = "[ ]";
+            parts = tasks[index].split(" ");//splits string into separate parts
+
+            if(parts[0].equals("done")) {
```

Comment:
Maybe use switch case to split up your task and use methods to handle the task required to keep the code clean